gaeguli/pipeline.c

```diff
@@ -164,10 +168,26 @@ gaeguli_pipeline_dispose (GObject * object)
         "GaeguliPipeline reference!");
   }
 
+  if (!self->loop || !g_main_loop_is_running (self->loop)) {
+    g_main_loop_quit (self->loop);
```

Comment:
If the loop is not running, you quit it? Shouldn't it be the other way around?

Reply:
It's a mistake while copy-n-paste.